EEBOS/owl-ui
# Chore/upgrade packages [sc-773]

### Short summary

- Updated a number of dependencies to match the versions in Scrowl.
- This caused Storybook and Parcel to break.
- Needed to upgrade Storybook to make proper use of webpack 5.
- Made a number of changes to the build:dist process
    - Brought in the custom plugins from Scrowl
    - Removed the "fixes" to parcel modules
    - Removed the static file asset relocater
    - standardized the build across all components, even the rollup `components` package
- Made some minor changes/fixes to the templates to reflect the new state of the build process

In general this PR brings us one step closer to having a global parcel config instead of individual configs for each package.

---

### Test steps

1. `yarn start:storybook` should function as expected
2. `yarn build:dist` should function as expected
3. `yarn create:component -n test` should function as expected
